zh/middleware/modules_communication.md

```diff
@@ -0,0 +1,117 @@
+# Modules Reference: Communication
```

Comment:
Shall I wait for this to be translated before I do a merge?

BTW, English version is auto generated from source but we won't do that for Chinese.